### DISCUSSION TITLE
code preview beta: wrong line number height

### DISCUSSION THREAD

**Discussion by j4k0xb** [npm/feedback#921] [Website] [answered]
I noticed that the line numbers don't have the same height as the lines, which accumulates the further you scroll down
![image](https://user-images.githubusercontent.com/55899582/235526374-0119d9e6-1a38-48d3-a3d0-974d9aac88b2.png)
![image](https://user-images.githubusercontent.com/55899582/235527047-4f1854bb-0c3d-41eb-b76d-38f2bf8fe002.png)

without the `line-height` rule it looks fine again
![image](https://user-images.githubusercontent.com/55899582/235553034-b41a3bc7-770e-4f78-8258-816dc19fd2ac.png)

**Comment by monishcm** [comment, marked as answer]
Feedback taken.

**Comment by arisacoba** [comment]
Hi, @j4k0xb. I can't replicate this for lengthy files. Is there a public file/package you can share where this happens? The line height for the numbers and the lines is currently the same, at 20px for both.

Which browser are you using?

**Reply by j4k0xb** [reply]
It happens on both the latest version of brave and firefox, mobile/desktop

https://www.npmjs.com/package/webcrack?activeTab=code file: `dist/cli.js`
